main.js

```diff
@@ -3,6 +3,13 @@
 const { app } = require("electron");
 const { start, lastWindow } = require("./index");
 
+// edge case when the program is launched without arguments
+if (process.argv.length == 1) {
+  app.requestSingleInstanceLock();
```

Comment:
Why don't you check the lock here ?

Reply:
cause there is nothing to do with the lock,
it only signals to the other window (if subscribed to the event).
then quitting the app cause there is no way the app can run without the host and port args configs

Reply:
ok makes sense